data/data/agent/files/usr/local/bin/pre-start-agent.sh.template

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ImagesEnvFile=${1}/agent-images.env
+
+IMAGE=`grep "AGENT_IMAGE" ${ImagesEnvFile}  | cut -d'=' -f2`
+
+echo "Using agent image: ${IMAGE} to copy bin"
+
+/usr/bin/podman run --privileged --rm -v /usr/local/bin:/hostbin --entrypoint='' ${IMAGE} cp /usr/bin/agent /hostbin
```

Comment:
Should be able to drop the `--entrypoint=''` now, the fix to the container is in both 4.11 and 4.12 nightlies.

Reply:
Fixed.